hck_setup.cfg

```diff
@@ -88,9 +88,9 @@ STUDIO_IMAGE=${IMAGES_DIR}/HCK_Studio_WS2008R2_SP1.qcow2
 CLIENT1_IMAGE=${IMAGES_DIR}/HCK_Client1_WS2008R2_SP1.qcow2
 CLIENT2_IMAGE=${IMAGES_DIR}/HCK_Client2_WS2008R2_SP1.qcow2
 
-#File System Testing Prerequisites image
-# If parameted defined and exists, the image will be copied from here,
-# otherwise it will be create a new unpartition one
+# File System Testing Prerequisites image
+# If this parameter is defined and the file exists, the test image will be
+# copied from it. Otherwise, a new unpartitioned image will be created.
```

Comment:
This is how I would have put it. Just didn't want to change too much previously. :)